### PR TITLE
Add target projects visualizations and correct package download data

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds visualizations for target projects and corrects package download data from PyPI (which was originally constrained due to default specifications). As part of this work I tried to prepare the existing web-focused content so the new content could be viewed from the same location without disrupting it.

Thank you for any thoughts or feedback you may have! Please don't hesitate to let me know if you have any questions.

### CODE REVIEW

File: docs/cytomining-ecosystem/index.html [hunk 1]
```diff
@@ -19,17 +19,23 @@ <h1>Cytomining Ecosystem Software Landscape Analysis</h1>
...
+           target ecosystem user base size, usage, and maturity.</p>
+        <ul>
+            <li><a href="report.html">Software Landscape Report</a></li>
+        </ul>
+
+        <h2>Landscape Data Profile</h2>
+        <p>A landscaep data profile (generated from <a
                 href="https://github.com/ydataai/ydata-profiling"><code>ydata-profiling</code></a>) is provided for
-            convenience below to help demonstrate further properties of the dataset used for the Focused Report.</p>
+            convenience below to help demonstrate further properties of the dataset used for the Lanscape Report.</p>
```
Comment (gwaybio) [inline docs/cytomining-ecosystem/index.html]:
```suggestion
            convenience below to help demonstrate further properties of the dataset used for the Landscape Report.</p>
```

File: docs/cytomining-ecosystem/index.html [hunk 2]
```diff
@@ -19,17 +19,23 @@ <h1>Cytomining Ecosystem Software Landscape Analysis</h1>
...
         <ul>
-            <li><a href="report.html">Focused Report</a></li>
+            <li><a href="target-project-report.html">Targeted Software Report</a></li>
         </ul>
 
-        <h2>Data Profile</h2>
-        <p>A data profile (generated from <a
+        <h2>Software Landscape Report</h2>
+        <p>A landscape report is provided below to help describe projects related to
+           target ecosystem user base size, usage, and maturity.</p>
```
Comment (gwaybio) [inline docs/cytomining-ecosystem/index.html]:
```suggestion
        <p>We provide a landscape report below to help describe projects related to
           our ecosystem's user base size, usage, and maturity.</p>
```

I'm not sure if my edits retain your original meaning!

**Review by gwaybio — APPROVED** [review]

File: docs/cytomining-ecosystem/index.html (hunk 1 above)
Comment (d33bs) [reply docs/cytomining-ecosystem/index.html]: Great catch, thanks!

File: docs/cytomining-ecosystem/index.html (hunk 2 above)
Comment (d33bs) [reply docs/cytomining-ecosystem/index.html]: Thank you for this, looks great!

**Comment by d33bs** [comment]
Thank you @gwaybio - merging this in!

**Pull request merged**